financial-qa-search/QAReranker/tests/test_qareranker.py

```diff
@@ -47,10 +47,11 @@ def test_qareranker():
     np.testing.assert_array_equal(
         new_scores,
         np.array(
-            [(1, 0.760756), (2, 0.00014822294)],
-            dtype=[(BaseRanker.COL_MATCH_HASH, np.int64), (BaseRanker.COL_SCORE, np.float64)],
+            [(1, 0.7607551217079163), (2, 0.0001482228108216077)],
```

Comment:
can u maybe have this test less wordy? I do not care about the dtypes here. Plus, try to have different tolerance, this values will break very easily with any small difference on package versions. Such a precision is not needed